sympy/integrals/manualintegrate.py

```diff
@@ -696,8 +697,13 @@ def trig_rule(integral):
         arg = integrand.args[0]
         rewritten = ((sympy.csc(arg)**2 + sympy.cot(arg) * sympy.csc(arg)) /
                      (sympy.csc(arg) + sympy.cot(arg)))
+    elif integrand == sympy.sqrt(1+sympy.sin(symbol)):
+        rewritten = (sympy.sin(symbol/2) + sympy.cos(symbol/2))
     else:
-        return
+        match = integrand.match(sympy.sqrt(1 + sympy.sin(a*symbol)))
+        if not match:
+            return
+        rewritten = (sympy.sin((match[a]*symbol)/2) + sympy.cos((match[a]*symbol)/2))
```

Comment:
This case is equivalent to the one above if a=1 so we don't really need two cases: this code can be simplified

Reply:
Ooohhh I just forgot to remove the above code.